packages/web/src/new/scenes/Main/Receive/ReceiveToken/Bitcoin/Hint/Hint.tsx

```diff
@@ -49,14 +53,25 @@ export const Hint: FC<Props> = ({ remainingTime }) => {
         </Row>
         <Row>
           <MinimumTxAmount>
-            Minimum transaction amount of &nbsp;
-            <strong>0.000112 BTC</strong>.
+            Minimum transaction amount of&nbsp;
+            {viewModel.isFetchingFee ? (
```

Comment:
it should be better if you make expr function with forming of this value (with condtion) outside jsx